### DISCUSSION TITLE
Is it possible to save a relationship from @formField('browser') by language?

### DISCUSSION THREAD

**Discussion by AndreiMsk** [area17/twill#950] [General]
Hello,

Could not find anything related in the documentation. Is it possible to attach a record (manyToMany) by language?

**Comment by pboivin** [comment]
Hi @AndreiMsk,

As far as I know, only the following field types are translatable: `input`, `wysiwyg`, `medias`, `files`.

For any other field type, you can implement translations by providing one field per language :

```
@section('contentFields')
    @formField('browser', [
        'moduleName' => 'banners',
        'name' => 'banner_en',
        'label' => 'Banner (EN)',
        'max' => 1,
    ])

    @formField('browser', [
        'moduleName' => 'banners',
        'name' => 'banner_fr',
        'label' => 'Banner (FR)',
        'max' => 1,
    ])
@stop
```

This is not ideal, but I think it's quite manageable for a few special fields. As usual, your mileage may vary :)

Here's a quick hack to hook into the language selector without any custom Vue component. This will only show the field for the currently selected language :

```
@section('contentFields')
    <div v-show="$root.currentLocale.value === 'en'">
        @formField('browser', [
            'moduleName' => 'banners',
            'name' => 'banner_en',
            'label' => 'Banner',
            'max' => 1,
        ])
    </div>

    <div v-show="$root.currentLocale.value === 'fr'">
        @formField('browser', [
            'moduleName' => 'banners',
            'name' => 'banner_fr',
            'label' => 'Banner',
            'max' => 1,
        ])
    </div>
@stop
```

Hope this helps!
